kornia/filters/sepia.py

```diff
@@ -0,0 +1,90 @@
+import torch
+import torch.nn as nn
+
+
+def sepia(input: torch.Tensor, rescale: bool = True) -> torch.Tensor:
+    r"""Apply to a tensor the sepia filter.
+
+    Args:
+        input: the input tensor with shape of :math:`(B, C, H, W)`.
+        rescale: If True, the output tensor will be rescaled (max values be 1. or 255).
+
+    Returns:
+        - torch.Tensor: The sepia tensor of same size and numbers of channels
+        as the input with shape :math:`(B, C, H, W)`.
+
+    Example:
+        >>> input = torch.ones(2, 3, 1, 1)
+        >>> sepia(input)
+        tensor([[[[1.0000]],
+                 [[0.8905]],
+                 [[0.6936]]],
+                [[[1.0000]],
+                 [[0.8905]],
+                 [[0.6936]]]])
+    """
+    if len(input.shape) < 3 or input.shape[-3] != 3:
```

Comment:
and below

Reply:
Ok, but this line https://github.com/kornia/kornia/blob/db988652b13f46ac0dfedc5f883e11a848a6ca07/kornia/testing/__init__.py#L279 shouldn't it also be `start_idx = 0` because this one (https://github.com/kornia/kornia/blob/db988652b13f46ac0dfedc5f883e11a848a6ca07/kornia/testing/__init__.py#L280) already cover it?

 If not, I didn't understand how it could be used for the tensor with shape ["*", "3", "H", "W"].